dev/bots/test.dart

```diff
@@ -879,11 +870,8 @@ Future<void> _stopChromeDriver() async {
   if (_chromeDriver == null) {
     return;
   }
+  print('Stopping chromedriver');
   _chromeDriver.process.kill();
-  while (await _isChromeDriverRunning()) {
```

Comment:
My assumption is that pinging the chromedriver while it's shutting down is what's triggering the issue in the client socket implementation, so removing the need to wait for chromedriver to shut down should also :crossed_fingers: fix the flakiness.